src/app/overview/overview.component.html

```diff
@@ -1,34 +1,117 @@
+    <div class="col-md-3 col-sm-6 mt-5"><h4>Incrementality</h4><p class=""> Instead of starting analysis from-scratch after a modification, Goblint is able to <b>reuse its results</b> for a previous version of the program and update
+        them to account for the effect of changes in a fraction of the time a from-scratch analysis takes.
     </p></div>
+    <!-- TODO: fill the feature descriptions
+    <div class="col-md-3 col-sm-6 mt-5"><h4>Starvation analysis</h4><p class=""> ...</p></div>
+    <div class="col-md-3 col-sm-6 mt-5"><h4>Incremental analysis of Git repositories</h4><p class="">Incremental analysis of Git repositories</p></div>
+    <div class="col-md-3 col-sm-6 mt-5"><h4>Fixpoint algorithms</h4><p class=""> Generic fixpoint algorithms with widening and narrowing.</p></div>
+    <div class="col-md-3 col-sm-6 mt-5"><h4>Global invariant</h4><p class=""> Global invariant approach and side-effecting.</p></div>
+    <div class="col-md-3 col-sm-6 mt-5"><h4>HTML front-end</h4><p class=""> Visualisation of results.</p></div> -->
+
 </div>
+<br />
+<br />
+<br />
 <div class="row d-flex justify-content-center">
-    <h2>Overview of Goblint Features</h2><br>
+    <h2>Supported Properties</h2><br>
 </div>
 <div class="row d-flex justify-content-center">
-    <div class="col-md-3 col-sm-6 mt-5"><h4>Race detection</h4><p class=""> The main application of this framework is for race detection. 
-        Goblint checks  that all accesses to shared memory are protected by a common mutex.</p></div>
-    <div class="col-md-3 col-sm-6 mt-5"><h4>Path-sensitivity</h4><p class=""> Goblint is path-sensitive, so it deals with conditional locking schemes as well as possibly failing, 
-        interruptible and probing locking primitives. Mutexes may be handled both pathsensitively and symbolically.</p></div>
-    <div class="col-md-3 col-sm-6 mt-5"><h4>Thread-modularity</h4><p class=""> Goblint performs thread-modular analysis of C programs. 
-        It computes a sound upper approximation of all possible thread interleavings using global invariants for shared data.</p></div>
-    <!-- TODO: fill the feature descriptions 
+    <div class="col-md-3 col-sm-6 mt-5"><h4>Race detection</h4><p class=""> Goblint started its life as a framework for race detection, and still excells here.
+        Goblint checks that all accesses to shared memory are protected by a common mutex, also taking into account which threads can execute in parallel, and which can't.</p></div>
+    <div class="col-md-3 col-sm-6 mt-5"><h4>Reach safety</h4><p class=""> Goblint is capable of checking whether some assert is guaranteed to hold at a given program point,
+        or whether a certain program is unreachable. This can be used to encode almost any safety property of interest.
```

Comment:
"Reach safety" is a weird term that's used for SV-COMP category names, but not much else I think. Maybe something like "Assertions and dead code" would be more tangible.

Also some of the wording is weird:
* "some assert" → "some assertion"?
* "a certain program is unreachable" → "a certain program _location_ is unreachable"?